t/basic.t

```diff
@@ -135,6 +135,18 @@ like ( $source_file->slurp,
        'Source file contains a GUID, as expected.',
 );
 
+### Test miscellaneous-attribute pass-through
+my $byline_post =
+    Path::Class::File->new(
+        $docroot_dir,
+        '2000-01-01-this-post-has-extra-headers.html',
```

Comment:
I'm probably misundertanding how things work, but shouldn't there be a corresponding file in `t/source_model`?